src/components/modal/modalsContainer.ts

```diff
@@ -0,0 +1,21 @@
+import Block from '../../scripts/block/block';
+import { Props } from '../../scripts/dto/types';
+
+export default class ModalsContainer extends Block {
+  constructor(props: Props) {
+    super({
+      tagName: props.tag || 'div'
+    }, 'div(data-child="modals notifications", class=\`modal-bg \${classList ? classList : ""}\`)', {
+      events: {
+        click: (e: Event) => {
+          const target: HTMLElement = e.target as HTMLElement;
+          if (!target.classList.contains('modal-bg')) return;
+          this.getContent().querySelectorAll('.modal-bg, .modal').forEach(item => item.classList.remove('opened'));
```

Comment:
У условия лучше сохранять блок с фигурными скобками, чтобы проще было читать (минификацию можно отдать соответствующей тулзе, а людям важна читаемость).

Конкретно здесь можно переделать условие
```suggestion
          if (target.classList.contains('modal-bg')) {
                    this.getContent().querySelectorAll('.modal-bg, .modal').forEach(item => item.classList.remove('opened'));
          }
```